### PR TITLE
Feature/highlight uncovered lines

### LINKED ISSUE #13: Highlighting of uncovered lines of code
# Feature request

As a user I would like to see uncovered lines of code highlighted so that,
- It's more obvious which lines need test coverage as well as 
- Uncovered lines can be easily found by scrolling through the minimap

Capability to toggle this feature can be incorporated by introducing an option in the package settings.

### PR DESCRIPTION
**Pull request by raiyankamal**
Resolves #13 

- Highlighting of uncovered lines of code in editor and minimap preview
- Option in package settings for toggling the highlight feature
- Demo screenshot added, docs updated

Screenshot can be seen in my branch: https://github.com/raiyankamal/PyCover/blob/feature/highlight-uncovered-lines/README.md

### CODE REVIEW

File: PyCover.sublime-settings [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
 	"python": "",
-	"onload": false
+	"onload": false,
+    "highlight_uncovered_lines": false
```
Comment (perimosocordiae) [inline PyCover.sublime-settings]: Nit pick: fix the indentation here.

File: README.md [hunk 2]
```diff
@@ -35,9 +35,15 @@ by default.
 You can also access it with the Command Palette under "Toggle Uncovered Lines",
 or via the right-click menu.
 
+### Settings
+
 With the settings option `"onload": true`,
 PyCover will start highlighting missing lines as soon as your file loads.
 
+The `highlight_uncovered_lines` option is used to toggle highlighting of uncovered lines. When this option is set to `true`, uncovered lines are highlighted in both editor window and minimap. However, this can be disruptive to the experience for some users, so by default this option is set to `false`.
```
Comment (perimosocordiae) [inline README.md]: Hard-wrap this line to ~80 columns, please.

**Review by perimosocordiae — APPROVED** [review]
Two minor comments, but this looks good to me.

**Comment by raiyankamal** [comment]
Thanks for the review. I made changes in response to your feedback. Please check.

**Comment by perimosocordiae** [comment]
Merged. Thanks again, @raiyankamal !

**Comment by raiyankamal** [comment]
You're welcome @perimosocordiae
I'm finding this plugin useful and may send a few more PRs as I discover the need to extend it.

Do you have any thoughts on when an updated release can be made? That'd make the updates available to general users via package control.

**Comment by perimosocordiae** [comment]
I'll cut a release now.

**Pull request merged**